Submitted By: Robert Connolly <robert at linuxfromscratch dot org> (ashes)
Date: 2005-02-18
Initial Package Version: 1.0.8
Upstream Status: Not submitted
Origin: http://svn.exactcode.de/t2/trunk/package/base/uclibc/patches/\
	alsa-utils-uclibc.patch
Description: This is needed to build alsa-utils with uClibc.

--- alsa-utils-1.0.8/alsamixer/alsamixer.c.orig	2005-02-03 05:24:41.000000000 +0000
+++ alsa-utils-1.0.8/alsamixer/alsamixer.c	2005-02-03 05:27:40.000000000 +0000
@@ -2188,7 +2188,7 @@
 mixer_signal_handler (int signal)
 {
   if (signal != SIGSEGV)
-    mixer_abort (ERR_SIGNAL, sys_siglist[signal], 0);
+    mixer_abort (ERR_SIGNAL, strsignal(signal), 0);
   else
     {
       fprintf (stderr, "\nSegmentation fault.\n");
